projects/ngx-location-viewer/src/lib/components/ngx-location-viewer.component.scss

```diff
@@ -22,10 +22,17 @@
     .whatishere-type {
         width: 4.0625rem;
     }
-    
+
     .whatishere-image {
         overflow: hidden;
     }
+
+    @media screen and (max-width: 410px) {
```

Comment:
I would handle this a bit more robust by making use of our provided mixins like this:
```
@include to($screen-xs-max) {
  .your-sidepanel {
    max-width: $navigation-menu-pane-width;
  }
}
```
I'm also wondering if it even needs to be in a media query. Isn't it better to have a single width for all sizes of the widget?

Tip 1: maybe you should take a look at the `.m-pane` class, instead of crafting your own one :-) https://github.com/a-ui/core_branding_scss/blob/master/src/styles/aui/_aui.pane.scss

Tip 2: I would provide an alternate close button inside the panel, like it is done in the modal: https://a-ui.github.io/core_branding_scss/#molecules-modal